rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,826 @@
+                    <link title="JAX-RS API" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/ext/ReaderInterceptor.html" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000122">
+            <when>
+                <javaclass references="org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataInputImpl.getFormData{*}">
+                    <location>METHOD_CALL</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Deprecated MultipartFormDataInputImpl#getFormData method" effort="1" severity="optional">
+                    <message>`MultipartFormDataInputImpl#getFormData` method will be removed in the future. Use `MultipartFormDataInput#getFormDataMap()` instead.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="Resteasy MultipartFormDataInputImpl#getFormData API" href="https://docs.jboss.org/resteasy/docs/3.0.13.Final/javadocs/org/jboss/resteasy/plugins/providers/multipart/MultipartFormDataInputImpl.html#getFormData()" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000123">
+            <when>
+                <javaclass references="org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataInput.getFormData{*}">
+                    <location>METHOD_CALL</location>
+                </javaclass>
+            </when>
+           <perform>
+                <hint title="Deprecated MultipartFormDataInput#getFormData method" effort="1" severity="optional">
+                    <message>`MultipartFormDataInput#getFormData` method will be removed in the future. Use `MultipartFormDataInput#getFormDataMap()` instead.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="Resteasy MultipartFormDataInput#getFormData API" href="https://docs.jboss.org/resteasy/docs/3.0.13.Final/javadocs/org/jboss/resteasy/plugins/providers/multipart/MultipartFormDataInput.html#getFormData()" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000124">
+            <when>
+                <javaclass references="Acme.Serve.Serve.AcmeSession.getSessionContext{*}">
```

Comment:
you are right, I just took everything what is deprecated from javadoc documentation, so there could be such cases.